### ISSUE TITLE
IntelliJ generator for Scala native lacking ill.scalanativelib dependency

### ISSUE THREAD

**Issue by mkotsbak** [lihaoyi/mill#602]
Importing the project into Intellij, the following packages are not found:

```
import mill.scalanativelib._
import mill.scalanativelib.api._
```

**Comment by andres-pipicello-olx** [comment]
Could you please provide a minimal working example? I would like to help but I am not versed in the Scala native world.

**Comment by lefou** [comment]
Looks like the marked duplicates of this issue were closed with #616 .
